src/Nethermind/Nethermind.Merge.Plugin/Handlers/V1/GetPayloadBodiesByRangeV1Handler.cs

```diff
@@ -0,0 +1,45 @@
+// SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Nethermind.Blockchain;
+using Nethermind.Core;
+using Nethermind.JsonRpc;
+using Nethermind.Logging;
+using Nethermind.Merge.Plugin.Data.V1;
+
+namespace Nethermind.Merge.Plugin.Handlers.V1;
+
+public class GetPayloadBodiesByRangeV1Handler : IGetPayloadBodiesByRangeV1Handler
+{
+    private readonly IBlockTree _blockTree;
+    private readonly ILogger _logger;
+
+    private const long MaxPayloadBodies = 256;
+
+    public GetPayloadBodiesByRangeV1Handler(IBlockTree blockTree, ILogManager logManager)
+    {
+        _blockTree = blockTree;
+        _logger = logManager.GetClassLogger();
+    }
+
+    public Task<ResultWrapper<ExecutionPayloadBodyV1Result?[]>> Handle(long start, long count)
+    {
+        if (count > MaxPayloadBodies)
+        {
+            if (_logger.IsInfo) _logger.Info($"{nameof(GetPayloadBodiesByRangeV1Handler)}. Too many payloads requested. Count: {count}");
+            return ResultWrapper<ExecutionPayloadBodyV1Result?[]>.Fail("Too many payloads requested",
+                ErrorCodes.LimitExceeded);
```

Comment:
Is this value from the spec? Shouldn't we have the same value for the second method?

Reply:
From spec: `Client software MUST support count values of at least 32 blocks.`

Reply:
Here they suggest 1024 as soft limit: https://github.com/ethereum/execution-apis/pull/218#issuecomment-1122493033